### PR TITLE
Add options for unlimited and forced widget resize

### LINKED ISSUE #2997: [FEATURE] Ability to resize widget any size
### Describe the feature

Example: Google Calendar schedule widget has 2x4 size and it is unable to change its size, especially the widget height while using Lawnchair launcher, and it really takes up space in the home screen. It is not limited to Google Calendar widget too, Todoist widget also having difficulties to adjust its widget height, where I'm unable to decrease the height.

### Why would it be useful to add?

To prevent the widget takes too much space in the home screen, and makes it obnoxious when user using their phone.

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

Screenshot_2022_10_03_16_07_53_59_67d7f8591eff9881972c1169b8bca2d8

### Additional context

_No response_

### LINKED ISSUE #3229: unlimited widget resize
### Describe the feature

in the old Lawnchair, any widget could be made any size, down to 1x1. now their size scales as the desktop dimensions do, and cannot be edited below the widget author's suggested minimum. I am requesting a return to the old behavior 

### Why would it be useful to add?

my homescreen is 99% widgets and it's hard to make them look right. allowing overlap helps, but still doesn't get all the way there 

### Did the feature exist in Lawnchair v2? (Play Store version)

Yes

### Example(s)

here's some additional text so that red star won't be angry

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by SolidTux**
## Description

This adds one option to force any widget to be resizeable and another one to remove the limit on widget sizes.

- Fixes #3229
- Fixes #2997

## Type of change

:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: lawnchair/res/values-de-rDE/strings.xml [hunk 1]
```diff
@@ -201,6 +201,8 @@
     <string name="color_dark">Dunkel</string>
     <string name="force_rounded_widgets">Abgerundete Ecken</string>
     <string name="allow_widget_overlap">Überlappen zulassen</string>
+    <string name="force_widget_resize">Alle Widgets skalieren</string>
+    <string name="widget_unlimited_size">Beliebige Größe für Widgets</string>
```
Comment (validcube) [inline lawnchair/res/values-de-rDE/strings.xml]: 👋 Head up, Crowdin CI will remove your changes on Sunday after this PR merged.

Consider translating the strings once this PR is merged at https://lawnchair.crowdin.com/lawnchair

Comment (SolidTux) [reply lawnchair/res/values-de-rDE/strings.xml]: Good to know, thank you! I intend to correct a mistake in the German translation anyway.

File: lawnchair/res/values/strings.xml [hunk 2]
```diff
@@ -231,6 +231,8 @@
 
     <string name="force_rounded_widgets">Rounded Corners</string>
     <string name="allow_widget_overlap">Allow Overlap</string>
+    <string name="force_widget_resize">Make all widgets resizable</string>
+    <string name="widget_unlimited_size">Unlimited widget size</string>
```
Comment (SuperDragonXD) [inline lawnchair/res/values/strings.xml]:
```suggestion
    <string name="force_widget_resize">Enforce Widget Resizing</string>
    <string name="widget_unlimited_size">Remove Size Constraints</string>
```

Lawnchair uses Title Case for most of its UI text. However, I'd suggest tweaking the text a bit more.

I'd also suggest adding a short description:
```
Enforce Widget Resizing
Allow resizing of widgets that are constrained to a specific size

Remove Size Constraints
Remove the minimum and maximum size restrictions of widgets
```

**Review by SuperDragonXD — COMMENTED** [review]
Aside from my suggested changes (and the comment regarding Crowdin), this PR looks good. I'm uncertain though on whether it can be merged before Lawnchair 14’s release.

File: lawnchair/res/values/strings.xml (hunk 2 above)
Comment (SolidTux) [reply lawnchair/res/values/strings.xml]: I'll implement this (hopefully later today).

File: lawnchair/res/values/strings.xml [hunk 3]
```diff
@@ -231,6 +231,10 @@
 
     <string name="force_rounded_widgets">Rounded Corners</string>
     <string name="allow_widget_overlap">Allow Overlap</string>
+    <string name="force_widget_resize_label">Enforce Widget Resizing</string>
+    <string name="force_widget_resize_description">Allow resizing of widgets that are constrained to a specific size</string>
+    <string name="widget_unlimited_size_label">Remove Size Constraints</string>
+    <string name="widget_unlimited_size_description                                             >Remove the minimum and maximum size restrictions of widgets</string>
```
Comment (SuperDragonXD) [inline lawnchair/res/values/strings.xml]:
```suggestion
    <string name="widget_unlimited_size_description">Remove the minimum and maximum size restrictions of widgets</string>
```

**Pull request merged**